omnibus/package-scripts/inspec/postinst

```diff
@@ -41,6 +41,6 @@ done
 # be manually fixed.
 chown -Rh 0:0 $INSTALLER_DIR
 
-echo "Thank you for installing InSpec!"
+echo "Thank you for installing Chef InSpec!"
```

Comment:
This is interesting to know - do we use this lingo elsewhere or is the party line to brand "Chef InSpec" where possible? (I support this change just interested in the background)